Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.5.3-slim-stretch
+
+WORKDIR /app
+COPY Gemfile* ./
```

Comment:
at this point in time, we don't have a `Gemfile` yet

Reply:
The gem file gets created with 'docker run' right?